README.md

```diff
@@ -255,17 +214,16 @@ If you like this project...
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-> Give credit to everyone who inspired your codebase.
 
-I would like to thank...
+
+I would like to thank Microverse community for giving me the opportunity to learn here.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- FAQ (optional) -->
 
 ## ❓ FAQ (OPTIONAL) <a name="faq"></a>
```

Comment:
[OPTIONAL] This section is optional 🤷🏽‍♀️  and if you don't want to edit it you may remove it to make the README look more professional. 🕴🏽